library/VTKExtensions/Rendering/vtkF3DRendererWithColoring.cxx

```diff
@@ -52,6 +52,7 @@ vtkSmartPointer<vtkTexture> GetTexture(const std::string& filePath, bool isSRGB
           texture->UseSRGBColorSpaceOn();
         }
         texture->InterpolateOn();
+        texture->SetColorModeToDirectScalars();
```

Comment:
Doesnt that impact all textures ?

Reply:
It does, but it was a bug related to #802